proto/interchain_security/ccv/provider/v1/query.proto

```diff
@@ -342,10 +343,33 @@ message QueryConsumerValidatorsValidator {
   // The power of the validator used on the consumer chain
   int64 power = 3;
   // The rate to charge delegators on the consumer chain, as a fraction
-  string rate = 4 [
+  string consumer_commission_rate = 4 [
     (gogoproto.customtype) = "cosmossdk.io/math.LegacyDec",
     (gogoproto.nullable)   = false
     ];
+  // description defines the description terms for the validator
+  cosmos.staking.v1beta1.Description description = 5 [(gogoproto.nullable) = false];
+  // operator_address defines the address of the validator's operator
+  string operator_address = 6 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+  // jailed defined whether the validator has been jailed from bonded status or not.
+  bool jailed = 7;
+  // status is the validator status (bonded/unbonding/unbonded).
+  cosmos.staking.v1beta1.BondStatus status = 8;
+  // The rate to charge delegators on the provider chain, as a fraction
+  string provider_commission_rate = 9 [
+    (gogoproto.customtype) = "cosmossdk.io/math.LegacyDec",
+    (gogoproto.nullable)   = false
+    ];
+  // provider_tokens defines the delegated tokens (incl. self-delegation).
+  string provider_tokens = 10 [
+    (cosmos_proto.scalar)  = "cosmos.Int",
+    (gogoproto.customtype) = "cosmossdk.io/math.Int",
+    (gogoproto.nullable)   = false
+  ];
+  // The power of the validator used on the provider chain
+  int64 provider_power = 11;
```

Comment:
Shall we deprecate the previously defined `power` and introduce the `consumer_power` field.

Reply:
That breaks backward compatibility, let's not.

Reply:
How exactly does it break backward compatibility?

Reply:
You are renaming a proto field while keeping the same enum index

Reply:
Please re-read my initial comment:
> Shall we deprecate the previously defined `power` and introduce the `consumer_power` field.

I never talked about renaming anything. I clearly said to deprecate `power` and introduce `consumer_power`. So, something like this:
```
// DEPRECATED blah blah ...
int64 power = 3 [deprecated = true];
...
int64 consumer_power = ...;
```
As far as I know, this is backwards compatible.

Reply:
Thanks for pointing it out, my appologies.

I have no preference if that's the way you want to go, as long as there's a field and it's not burried in a nested JSON - great 😄